package.json

```diff
@@ -12,6 +12,8 @@
     "test": "yarn build && jest --no-cache",
     "test:watch": "yarn build && jest --no-cache --watch",
     "test:cover": "yarn test --collectCoverage",
+    "lint": "eslint --config .eslintrc.js --ext tsx,ts ./packages/**/src ./examples",
+    "lint:fix": "yarn lint -- --fix",
```

Comment:
can we add a pre-commit hook for this? + prettier

Reply:
yeah probably, i'll have a look

Reply:
yeah not sure how that would play with pretty-quick 🤔

Reply:
nevermind, should be fine to call it after 👍 added